validator/errors/errors.py

```diff
@@ -0,0 +1,22 @@
+"""Custom Errors for Nestor Config Validator"""
+
+
+class InvalidTargetPathError(Exception):
+    """Exception raised when the target path (NESTOR_TARGET_PATH) env is not set"""
+
+    def __init__(self, target_path: str):
+        super().__init__(
+            f"The environment variable NESTOR_TARGET_PATH is not set. Current value: ${target_path}"
+        )
+
+
+class ConfigurationNotValidError(Exception):
```

Comment:
Is this error class used ? (or will it be used later ?)

Reply:
It will be used later. I'm aiming to enhance the way we report errors.